server.js

```diff
@@ -41,6 +41,9 @@ module.exports = !WebSocket.Server ? null : function (opts, onConnection) {
   wsServer.on('connection', function (socket, req) {
     var stream = ws(socket)
     stream.remoteAddress = req.socket.remoteAddress
+    stream.remotePort = req.socket.remotePort
+    stream.close = () => emitter.close()
```

Comment:
Shouldn't `emitter.close()` be `socket.close()`? `emitter.close` is going to shut the whole server down, not close the connection.

I'd also map `stream.destroy` to `socket.terminate` for consistency with the returned `client` streams.